automated-comments/go/two-fer/strings_builder_used_for_concatenation.md

```diff
@@ -0,0 +1,6 @@
+`strings.Builder` is a highly optimized way to build complex strings.
+In simple cases like this using `fmt.Sprintf` is the preferred way. It has a lot
+of formatting options making it very flexible far beyond simple concatenation.
+
+The `+` operator is a very fast alternative for concatenations.
```

Comment:
This is giving too many options.

I don't think _speed_ in `two-fer` is a concern at all. Another suggestion or idea:

- accept stringbuilder, you MAY add a "did you know about sprintf"
- accept sprintf, you MAY add "in simple use cases, + is a great alternative for concatenations"
- accept +, don't say anything. Instead, next time if they use + when it wasn't a good choice, tell them, then.

Reply:
Yeah, I agree that too many options is not great at the start of the process. I would agree with mentioning `fmt` and accepting `+` without comment.

Reply:
Probably a good idea to provide less options, yes. I'll remove the `+` part.